CodeGeneration/Sources/SyntaxSupport/KeywordSpec.swift

```diff
@@ -274,6 +274,7 @@ public enum Keyword: CaseIterable {
   case `typealias`
   case unavailable
   case unchecked
+  case preconcurrency
```

Comment:
Sorry if this is nitpicking, but this list seems to be sorted - shouldn't `preconcurrency` go higher up?

Reply:
I didn't know that it was supposed to be sorted but I can move it.

Reply:
Right, I just missed the comment at the very top...